tests/e2e/pubsub/jetstream/docker-compose.yaml

```diff
@@ -9,7 +9,7 @@ services:
       - ./jwt:/jwt
 
   dev:
-    image: golang:1.17.6
+    image: golang:1.18
```

Comment:
Should pin a specific version?

Reply:
This pins to 1.18 but allows patch updates.

Reply:
@daixiang0 this should be ok -- patch version changes are fine